### ISSUE TITLE
MPlayer OSX Extended crashing on High Sierra?

### ISSUE THREAD

**Issue by taylornicole** [sttz/MPlayer-OSX-Extended#89]
I updated my macbook to high sierra and now MPlayer OSX Extended keeps crashing (below is the details that it gives me when it crashes) Is anyone else having this issue? If so, has anyone found a way to fix this? Thank you in advance!

**problem details and system configuration that i get when it crashes:**

Process:               MPlayer OSX Extended [3505]
Path:                  /Applications/MPlayer OSX Extended.app/Contents/MacOS/MPlayer OSX Extended
Identifier:            hu.mplayerhq.mplayerosx.extended
Version:               rev15 (1511)
Code Type:             X86-64 (Native)
Parent Process:        ??? [1]
Responsible:           MPlayer OSX Extended [3505]
User ID:               501

Date/Time:             2017-10-05 16:58:25.977 -0400
OS Version:            Mac OS X 10.13 (17A365)
Report Version:        12
Anonymous UUID:        893EBB3B-3CAD-8ECC-975B-3507DBD19457

Sleep/Wake UUID:       EE103052-561B-4352-B1CE-3E6FDE11226F

Time Awake Since Boot: 13000 seconds
Time Since Wake:       4600 seconds

System Integrity Protection: enabled

Crashed Thread:        0  Dispatch queue: com.apple.main-thread

Exception Type:        EXC_CRASH (SIGABRT)
Exception Codes:       0x0000000000000000, 0x0000000000000000
Exception Note:        EXC_CORPSE_NOTIFY

Application Specific Information:
*** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'Cannot update for observer <NSUserDefaultsController 0x6000000e2980> for the key path "MPECustomAspectRatio.MPECustomAspectRatioString" from <NSUserDefaults 0x6000000400c0>, most likely because the value for the key "MPECustomAspectRatio" has changed without an appropriate KVO notification being sent. Check the KVO-compliance of the NSUserDefaults class.'
terminating with uncaught exception of type NSException

**Comment by GlawenClattuck** [comment]
Hi dude, your request have an answer. Look inside the post "Crash on High Sierra" and thanks to kokoman1, we all have a workaround.

**Comment by pdaugavietis** [comment]
See https://github.com/sttz/MPlayer-OSX-Extended/issues/88

**Comment by Availebla** [comment]
I have a recent problem on MplayerX since updating to High Sierra. Any video file I open is no longer filling the screen (in full screen mode) i.e. the video plays inside a black square. Toggling fill screen works but crops embedded subtitle files. I did not have this issue before as MplayerX would fill the screen by default to the maximum without cropping anything. Please advise in layman's terms if possible. I have tried tweaking the settings under preferences but none of them match the problem I have described. I have also tried uninstalling and reinstalling MplayerX several times but the problem persists. (I deleted all the files form ~Library/Preferences , Cache and everywhere else before each reinstall but to no avail

**Comment by sttz** [comment]
Fixed in rev16-test2:
https://github.com/sttz/MPlayer-OSX-Extended/releases/tag/rev16-test2
